nettacker/modules/vuln/wp_plugin_cve_2021_39316.yaml

```
@@ -2,49 +2,52 @@ info:
   name: wp_plugin_cve_2021_39316_vuln
   author: OWASP Nettacker Team
   severity: 7
-  description: The Zoomsounds plugin <= 6.45 for WordPress allows arbitrary files, including sensitive configuration files such as wp-config.php, to be downloaded via the `dzsap_download` action using directory traversal in the `link` parameter.
-  reference: 
-    - https://wpscan.com/vulnerability/d2d60cf7-e4d3-42b6-8dfe-7809f87547bd
-    - https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-39316
+  description: The Zoomsounds plugin <= 6.45 for WordPress allows arbitrary 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)